OctoPrint/OctoPrint
# Fix - First Run UI never loads due to missing jinja references

#### What does this PR do and why is it necessary?

This PR fixes the initial setup wizard by removing stray references to recently removed jinja snippets.

The recent transition to `Classic Webcam` (#4628) left some references to old jinja snippets in the initial setup wizard, this caused the template rendering to fail and the OctoPrint UI to never load.

#### How was it tested? How can it be tested by the reviewer?

- Start an OctoPrint instance based on the latest `maintenance` branch (make sure the initial setup wizard will show up).
- Wait forever because it won't load.
- Refresh and observe `Internal Server Error`.
  - See Error Traceback below

<details>
<summary>Error Traceback</summary>

```
2023-01-25 22:34:44,295 - octoprint - ERROR - Exception on / [GET]
Traceback (most recent call last):
  File "/Users/kestin/Projects/OctoPrint/venv/lib/python3.8/site-packages/flask/app.py", line 2525, in wsgi_app
    response = self.full_dispatch_request()
  File "/Users/kestin/Projects/OctoPrint/venv/lib/python3.8/site-packages/flask/app.py", line 1822, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/Users/kestin/Projects/OctoPrint/venv/lib/python3.8/site-packages/flask/app.py", line 1820, in full_dispatch_request
    rv = self.dispatch_request()
  File "/Users/kestin/Projects/OctoPrint/venv/lib/python3.8/site-packages/flask/app.py", line 1796, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)
  File "/Users/kestin/Projects/OctoPrint/src/octoprint/server/views.py", line 714, in index
    response = default_view()
  File "/Users/kestin/Projects/OctoPrint/src/octoprint/server/views.py", line 646, in default_view
    return preemptively_cached()
  File "/Users/kestin/Projects/OctoPrint/src/octoprint/server/util/flask.py", line 1191, in decorated_function
    return f(*args, **kwargs)
  File "/Users/kestin/Projects/OctoPrint/src/octoprint/server/util/flask.py", line 1273, in decorated_function
    return f(*args, **kwargs)
  File "/Users/kestin/Projects/OctoPrint/src/octoprint/server/util/flask.py", line 1283, in decorated_function
    r = f(*args, **kwargs)
  File "/Users/kestin/Projects/OctoPrint/src/octoprint/server/util/flask.py", line 929, in decorated_function
    rv = f_with_duration(*args, **kwargs)
  File "/Users/kestin/Projects/OctoPrint/src/octoprint/server/util/flask.py", line 881, in f_with_duration
    return f(*args, **kwargs)
  File "/Users/kestin/Projects/OctoPrint/src/octoprint/server/util/flask.py", line 1202, in decorated_function
    rv = f(*args, **kwargs)
  File "/Users/kestin/Projects/OctoPrint/src/octoprint/server/util/flask.py", line 1227, in decorated_function
    rv = f(*args, **kwargs)
  File "/Users/kestin/Projects/OctoPrint/src/octoprint/server/views.py", line 636, in make_default_ui
    r = make_response(render_template("index.jinja2", **render_kwargs))
  File "/Users/kestin/Projects/OctoPrint/venv/lib/python3.8/site-packages/flask/templating.py", line 147, in render_template
    return _render(app, template, context)
  File "/Users/kestin/Projects/OctoPrint/venv/lib/python3.8/site-packages/flask/templating.py", line 130, in _render
    rv = template.render(context)
  File "/Users/kestin/Projects/OctoPrint/venv/lib/python3.8/site-packages/jinja2/environment.py", line 1301, in render
    self.environment.handle_exception()
  File "/Users/kestin/Projects/OctoPrint/venv/lib/python3.8/site-packages/jinja2/environment.py", line 936, in handle_exception
    raise rewrite_traceback_stack(source=source)
  File "/Users/kestin/Projects/OctoPrint/src/octoprint/templates/index.jinja2", line 172, in top-level template code
    {% include 'dialogs/wizard.jinja2' %}
  File "/Users/kestin/Projects/OctoPrint/src/octoprint/templates/dialogs/wizard.jinja2", line 45, in top-level template code
    {% include data.template ignore missing %}
  File "/Users/kestin/Projects/OctoPrint/src/octoprint/plugins/corewizard/templates/corewizard_webcam_wizard.jinja2", line 29, in top-level template code
    {% include "snippets/settings/webcam/webcamStreamUrl.jinja2" %}
  File "/Users/kestin/Projects/OctoPrint/venv/lib/python3.8/site-packages/flask/templating.py", line 62, in get_source
    return self._get_source_fast(environment, template)
  File "/Users/kestin/Projects/OctoPrint/venv/lib/python3.8/site-packages/flask/templating.py", line 98, in _get_source_fast
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: snippets/settings/webcam/webcamStreamUrl.jinja2
```
</details>

#### What are the relevant tickets if any?

None that I'm aware of.

#### Screenshots (if appropriate)

![Screen Shot 2023-01-25 at 10 34 56 PM](https://user-images.githubusercontent.com/7365747/214754197-5cef4726-1b2f-4863-9d55-0fd4fb073337.png)

#### Further notes

Removing the snippet references caused 2 'webcam' pages in the wizard, removing the `WebcamSubwizard` class fixes this.

| Before | After |
|:-:|:-:|
|![Before](https://user-images.githubusercontent.com/7365747/214754363-5fac3822-2589-4d95-babd-a241705c34a0.png)|![After](https://user-images.githubusercontent.com/7365747/214754237-d3d656c1-13f5-4b93-9eb7-b87dec5680f6.png)|
